src/main/scala/com/lightbend/kafkalagexporter/PrometheusEndpointSink.scala

```diff
@@ -69,4 +66,13 @@ class PrometheusEndpointSink private(definitions: MetricDefinitions, metricWhite
     registry.clear()
     server.stop()
   }
+
+  def globalLabelNames(): List[String] = {
+    clusterGlobalLabels.values.flatMap(_.keys).toList.distinct
+  }
+
+  def getGlobalLabelNames(clusterName: ClusterName): List[String] = {
```

Comment:
This is returning label values, not names, for a cluster.  Maybe it could be called `getGlobalLabelValuesOrDefault`.